src/Traversal/Sdk/Sdk.targets

```diff
@@ -111,47 +126,67 @@
     <MSBuild Projects="@(ProjectReference)"
              BuildInParallel="$(BuildInParallel)"
              SkipNonexistentProjects="$(SkipNonexistentProjects)"
-             SkipNonexistentTargets="$(SkipNonexistentTargets)" />
+             SkipNonexistentTargets="$(SkipNonexistentTargets)"
+             ContinueOnError="$(ContinueOnError)" />
+    
+    <!-- If we ErrorAndContinue we need to propagate the error. -->
+    <Error Condition="'$(ContinueOnError)' == 'ErrorAndContinue' and '$(MSBuildLastTaskResult)' == 'false'" />
```

Comment:
I have never seen this pattern before.  Doesn't ErrorAndContinue mean to still treat the task as an error but then continue executing tasks in the target?  But then we call `<Error />` with no message in that case? I'm assuming there's a logical explanation, can you fill me in? 😄

Reply:
I noticed that in dotnet/runtime, I think that came from here: https://github.com/dotnet/corefx/commit/60967f18fd7b47dff0576759e3ea6dcb1ea3cbcb

Reply:
ok as discussed offline, unnecessary.